tests/tit/core/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ add_tit_test_from_target(
 add_tit_executable(
   NAME core_catch_ctrl_c_test
   SOURCES "catch_ctrl_c.cpp"
-  DEPENDS tit::core)
```

Comment:
There is circular dependency between `core` and `par` at the moment. We need to break it.